scripts/release/get_last_release.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o pipefail
+
+GITHUB_URL=https://api.github.com/repos/${CODE_REPOSITORY}
+
+CURL_RESPONSE=$(curl -L \
+  -s \
+  --fail-with-body \
+  -H "Accept: application/vnd.github+json" \
+  -H "X-GitHub-Api-Version: 2022-11-28" \
+  "${GITHUB_URL}"/releases/latest) 
```

Comment:
I agree with your comment.
I used [shellcheck](https://www.shellcheck.net/) for all the scripts and I removed the optional curly braces.
As for the  `"${GITHUB_URL}"/releases/latest` vs `"${GITHUB_URL}/releases/latest"`, I prefer quoting only the variable part, because this is what the shellcheck suggests:
```
In get_last_release.sh line 12:
  $GITHUB_URL/releases/latest)
  ^---------^ SC2086 (info): Double quote to prevent globbing and word splitting.
```